lib/xcpretty/parser.rb

```diff
@@ -64,6 +82,23 @@ module Matchers
     # $2 file_path
     COMPILE_COMMAND_MATCHER = /^\s*(.*clang\s.*\s\-c\s(.*\.(?:m|mm|c|cc|cpp|cxx))\s.*\.o)$/.freeze
 
+    # @regex Captured groups
+    COMPILE_SWIFT_SOURCES_MATCHER = /^CompileSwiftSources\s.*$/.freeze
+
+    # @regex Captured groups
+    COMPILE_SWIFT_MATCHER = /^CompileSwift\s.*$/.freeze
+
+    # @regex Captured groups
+    SWIFT_CODE_GENERATION_MATCHER = /^SwiftCodeGeneration\s.*$/.freeze
```

Comment:
Consider using state of build_target, then you can get rid of this regex altogether.

Reply:
Done